### DISCUSSION TITLE
Error when deploying in npm.php

### DISCUSSION THREAD

**Discussion by vbasky** [deployphp/deployer#2894] [Help needed] [answered]
- Deployer version: 7.0.0-rc.3
- Deployment OS:  Linux Ubuntu

I am linking the issue that I raised in https://github.com/lorisleiva/laravel-deployer repo

[Laravel Deployer](https://github.com/lorisleiva/laravel-deployer/issues/247)

Has something changed in the npm.php file in the latest RC. It used to be working previously don't remember the RC version

**Comment by mattkibbler** [comment, marked as answer]
I had the same issue and it turned out to be a problem with NVM ([as mentioned here](https://github.com/deployphp/deployer/issues/2535)). 

Maybe that could be your problem?

**Reply by vbasky** [reply]
I confirm it indeed was a problem with NVM. I removed NVM and installed Node 14. I am now able to deploy using deployer
